tasks/components/TaskTile.tsx

```diff
@@ -0,0 +1,36 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { CardProps } from './Card'
+import { Card } from './Card'
+import { ProgressIndicator } from './ProgressIndicator'
+
+type TaskDTO = {
+  name: string,
+  description: string
+}
+
+export type TaskTileProps = CardProps & {
+  progress: number,
+  task: TaskDTO
+}
+
+export const TaskTile = ({
+  progress,
+  task,
+  isSelected = false,
+  onTileClick = () => undefined
+}: TaskTileProps) => {
+  return (
+    <Card onTileClick={onTileClick} isSelected={isSelected}>
+      <div className={tw('flex flex-row justify-between w-full')}>
+        <div className={tw('overflow-hidden')}>
+          <span className={tw('font-bold font-space')}>{task.name}</span>
+          <br/>
```

Comment:
Wrap the two `span`\`s into a `<div className="flex flex-col"> ... </div>` instead? Or is this considered an overuse of Flexbox? 😅 Idk